libs/src/services/creatorNode/CreatorNode.ts

```diff
@@ -765,6 +769,11 @@ export class CreatorNode {
 
       axiosRequestObj.headers = axiosRequestObj.headers || {}
 
+      // Stringify `writeQuorumEnabled` to work around axios bug (fixed in axios v0.20.0): https://github.com/axios/axios/issues/2223
+      axiosRequestObj.headers['Enforce-Write-Quorum'] = String(
+        this.writeQuorumEnabled
+      )
```

Comment:
I was trying to have some neutral default so the env var could control it if the client doesn't have a preference, but yeah it actually now defaults to 'false' anyway because the Optimizely flag only allows true/false. shouldn't be an issue either way now, although on the CN side we should support `undefined` since some clients won't send the param yet

Reply:
agreed on supporting this client side, just not sure it makes sense to default to non boolean on client side
don't think its an issue as is, just bit more complexity - your call

Reply:
yeah I agree I'm not seeing any reason to default to null anymore. I'll change it and re-test before merging. I still have to fix all the tests that are failing anyway